ospf6d/ospf6_lsdb.h

```diff
@@ -25,10 +25,25 @@
 #include "table.h"
 #include "ospf6_route.h"
 
+/* LSA Statistics */
+struct ospf6_lsdb_stats {
+	uint32_t num_type1; /* Router-LSA Count */
+	uint32_t num_type2; /* Network-LSA Count */
+	uint32_t num_type3; /* Inter-Area-Prefix-LSA Count */
+	uint32_t num_type4; /* Inter-Area-Router-LSA Count */
+	uint32_t num_type5; /* AS-External-LSA Count */
+	uint32_t num_type7; /* NSSA-LSA Count */
+	uint32_t num_type8; /* Link-LSA Count */
+	uint32_t num_type9; /* Intra-Area-Prefix-LSA Count */
+};
+
+#define OSPF6_LSDB_STATS_SUB(stats, type)
+
 struct ospf6_lsdb {
 	void *data; /* data structure that holds this lsdb */
 	struct route_table *table;
 	uint32_t count;
+	struct ospf6_lsdb_stats stats; /* LSA statistics */
```

Comment:
Please put the comment above and not at the end

Reply:
fixed.